### DISCUSSION TITLE
Refreshing Followers List Duplicates the List

### DISCUSSION THREAD

**Discussion by GovindKrishnan** [community/community#12881] [Mobile] [answered]
On GitHub Mobile, if you drag & refresh someone's followers or following list from profile, it gets duplicated by each refresh.

**Comment by aconsuegra** [comment, marked as answer]
Thanks for the report! We have been able to reproduce and fix the issue so it will be fixed in our upcoming release.

**Reply by GovindKrishnan** [reply]
Great! Thanks! :octocat: ❤️

**Reply by GovindKrishnan** [reply]
Just got this update. Not sure if you were referring to this version as the issue still exists.

![Screenshot_2022-03-18-21-54-25-95_b5a5c5cb02ca09c784c5d88160e2ec24.jpg](https://user-images.githubusercontent.com/74248475/159042752-772781fc-2b37-4733-b4aa-15e7ab9f609d.jpg)

**Reply by aconsuegra** [reply]
No, the fix is included in version 1.57, which is already available for beta testers. It will be released for everyone next week.

**Reply by zhouhuandev** [reply]
发现关注人与被关注人列表刷新会产生多次重复加载数据问题。非常感谢贵团队的付出，期待修复过的版本。
